src/lib/file.js

```diff
@@ -77,6 +77,18 @@ export function checkPermission(file, operation) {
     return !!file && !!file.permissions && !!file.permissions[operation];
 }
 
+/**
+ * Checks permission
+ *
+ * @public
+ * @param {Object} file - Box file
+ * @param {string} operation - Action to check permission for
+ * @return {boolean} Whether or not action is permitted
+ */
+export function canDownloadWatermarkedRep(file) {
```

Comment:
This is temporary, Albert will be provided a `can_download_watermarked` boolean to preview that accurately dictates if we can download the watermarked rep

Reply:
👍